connector/src/main/java/org/odpi/egeria/connectors/juxt/xtdb/repositoryconnector/XtdbOMRSRepositoryConnector.java

```diff
@@ -366,6 +367,38 @@ public EntityDetail getResultingEntity(String docId,
         }
     }
 
+    /**
+     * Validates that the commit was persisted (if synchronous), throwing an exception if it failed, and
+     * also retrieves and returns the summary entity that resulted from the transaction. Note that if the
+     * operation is configured to be asynchronous, this will ALWAYS return null for the entity details.
+     * @param docId of the entity within XTDB itself (i.e. prefixed)
+     * @param instant giving the commit point of the transaction
+     * @param methodName that made the commit
+     * @return EntitySummary result of the committed transaction (synchronous) or null (asynchronous)
+     * @throws Exception on any error
+     */
+    public EntitySummary getResultingEntitySummary(String docId,
+                                                   TransactionInstant instant,
+                                                   String methodName) throws Exception {
+        validateCommit(instant, methodName);
```

Comment:
If just about anything is called before the `.start()` of the connector, all kinds of things will go wrong... I get the general suggestion, but this doesn't really apply in this case since the startup sequence of connectors ensures that `.start()` is called before any other method would be.